plugins/node/opentelemetry-instrumentation-mongodb/src/types.ts

```diff
@@ -16,6 +16,7 @@
 
 import { InstrumentationConfig } from '@opentelemetry/instrumentation';
 import { Span } from '@opentelemetry/api';
+import * as mongodb from 'mongodb';
```

Comment:
You cannot import from your instrumented library (mongodb)
This will crash applications that installed the instrumentation but not `mongodb` package (valid use case).
Use `import type * as mongodb from 'mongodb';`
This will remove the import fromo JS after transpilation.

Reply:
Good one! fixed